spearhead/search.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * The template for displaying search results pages
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/#search-result
+ *
+ * @package Spearhead
+ * @since 1.0.0
+ */
+
+get_header();
+?>
+
+	<section id="primary" class="content-area">
+		<main id="main" class="site-main" role="main">
+
+		<?php if ( have_posts() ) : ?>
+
+			<?php get_search_form(); ?>
+
+			<?php
+			// Start the Loop.
+			while ( have_posts() ) :
+				the_post();
+				?>
+
+				<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+					<header class="entry-header default-max-width">
+						<?php
+						if ( is_sticky() && is_home() && ! is_paged() ) {
+							printf( '<span class="sticky-post">%s</span>', _x( 'Featured', 'post', 'seedlet' ) );
+						}
+						the_title( sprintf( '<h2 class="entry-title"><a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a></h2>' );
+						?>
+					</header><!-- .entry-header -->
+
+					<footer class="entry-footer default-max-width">
+						<?php spearhead_entry_meta_footer(); ?>
```

Comment:
I'm getting an error here unless I change this to `seedlet_entry_meta_footer`. Is that correct, or should the filter functions be renamed instead?